.travis.yml

```diff
@@ -0,0 +1,24 @@
+dist: trusty
+sudo: false
+
+language: node_js
+node_js:
+  - '10'
```

Comment:
Should we use node 12 here i.e. the same as in the Dockerfile?

Reply:
Bumped version up to 12.